README.md

```diff
@@ -49,7 +53,7 @@ The following example is tested on a minikube, so that you can play with this op
 % cd "$GOPATH"/src/github.com/open-cluster-management/multicloud-operators-channel
 ```
 
-- Setup environment and deploy channel operator
+2. Set up the environment and deploy the channel operator:
 
 ```shell
 # apply all the necessary CRDs
```

Comment:
Tech doc is more strict, open is looser. In my opinion, we can take the comment and add it to the actual step to clean this up:

Run the following command to apply all necessary CRDs. I cannot make you do that, it's your repo, lol, but recommend we use these comments as part of the steps for the user. :-)